linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# platform = multi_platform_sle
```

Comment:
Please add the metadata to this file.
```
# reboot = false
# strategy = restrict
# complexity = low
# disruption = low
```

Reply:
Hello @Mab879 - I had done what you proposed.